rangeChecker.Tests/RangePrintInCsvTests.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections.Generic;
+using Xunit;
+
+namespace Range.Tests
+{
+    public class RangePrintInCsvTests
+    {
+        void TestRange(List<int> values, string expectedOutput)
+        {
+            string outputFormat = "{0}-{1}, {2}\n";
+            string actualOutput = RangeChecker.GetContinuousRangesInCsv(values, outputFormat);
+            Assert.Equal(expectedOutput, actualOutput);
+        }
+
+        [Fact]
+        void TestRegularValues()
+        {

```

Comment:
Good set of test cases :+1: 
You have assumed that single value range should be ignored and that assumption is captured as a requirement in the tests, well done !!